_gsocprojects/2019/project_AllenSIMTtoSPMD.md

```diff
@@ -0,0 +1,10 @@
+---
+title: SIMT to SPMD code translation
+project: Allen
+layout: gsoc_proposal
```

Comment:
@dcampora Project descriptions do not use this layout, it should be `default` instead:

```suggestion
layout: default
```